playground/index.ts

```diff
@@ -1,18 +1,52 @@
-
-import * as moltin from './../';
+import * as moltin from '..'
 
 async function main() {
   const gateway = moltin.gateway({
     host: 'epcc-integration.global.ssl.fastly.net',
     client_id: 'client_id',
     client_secret: 'client_secret',
     headers: {
-      'EP-Beta-Features': 'account-management',
-    },
-  });
+      'EP-Beta-Features': 'account-management,accounts'
+    }
+  })
+  // await gateway.Authenticate()
 
-  // const accounts = await gateway.Accounts.All();
-  // console.log(accounts);
-}
+  // await gateway.Accounts.Create({
```

Comment:
Can you please remove playground from PR

Reply:
done